classes/patterns/class-patterns.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Automattic\Gravatar\GravatarEnhanced\Patterns;
+
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
Personally I don't like these at the tops of files, and we don't use them anywhere else.